tests/js/searchtools.js

```diff
@@ -0,0 +1,32 @@
+describe('Basic html theme search', function() {
+
+  describe('terms search', function() {
+
+    it('should find "C++" when in index', function() {
+      index = {
+        docnames:["index"],
+        filenames:["index.rst"],
+        terms:{'c++':0},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/angrycaptain19/sphinx/pullRequest?prid=7480054)
```suggestion
        terms:{"c++":0},
```